WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/ShippingLabelPackageDetailsViewModel.swift

```diff
@@ -41,13 +41,16 @@ final class ShippingLabelPackageDetailsViewModel: ObservableObject {
     @Published var selectedPackageID: String?
 
     /// List of selected package with basic info.
-    /// TODO-4599: update this to properly support multi-package.
+    /// This is a workaround to work with multi-package solution.
```

Comment:
nit: you didn't explain in the comment what is the workaround. It could be useful in my opinion for the future.

Reply:
Thanks for spotting this. I've updated the comment in 8fd357897aec61505fa569205fa04ae5f1cb448d.